### PR TITLE
Add `compose-ui` playground.

### PR DESCRIPTION
**Pull request by tikurahul**
* Added support for NDK builds.

Test: `./gradlew test`

### CODE REVIEW

File: .github/workflows/presubmit.yml [hunk 1]
```diff
@@ -62,6 +62,7 @@ jobs:
         shell: bash
         run: |
           set -x
+          echo "ANDROID_NDK_HOME=$ANDROID_NDK_LATEST_HOME" >> $GITHUB_ENV
```
Comment (liutikas) [inline .github/workflows/presubmit.yml:66]: Please add a comment that we need NDK 23 and that default ANDROID_NDK_HOME is 21.

https://github.com/actions/virtual-environments/blob/4f8049546ee84b94b4fe947bab7fa9558b34b195/images/linux/Ubuntu2004-README.md#environment-variables-3

**Review by liutikas — COMMENTED** [review]
Should we use project() for all the dependencies that we know are within compose playground as discussed in chat?

File: .github/workflows/presubmit.yml (hunk 1 above)
Comment (tikurahul) [reply .github/workflows/presubmit.yml:66]: Done.

**Comment by tikurahul** [comment]
Used `project()` consistently.

**Pull request closed without merging**